### PR TITLE
Add @emotion/core/macro, remove @emotion/css

### PR DESCRIPTION
**Pull request by Andarist**

### CODE REVIEW

**Comment by Andarist** [comment]
@mitchellhamilton besides changesets - this is pretty much ready to be reviewed.

File: docs/babel-macros.mdx [hunk 1]
```diff
@@ -4,11 +4,11 @@ title: 'Babel Macros'
 
 [Create React App recently added support for Babel Macros](https://reactjs.org/blog/2018/10/01/create-react-app-v2.html) which makes it possible to run Babel transforms without changing a Babel config.
 
-Most of Emotion's Babel Macros are available by adding `/macro` to the end of the import you'd normally use. (assuming you're using Create React App v2 or you've added babel-plugin-macros to your .babelrc) For example, to use the macro for styled, use `@emotion/styled/macro`. The one exception to this is `@emotion/core`, `@emotion/core` doesn't have a Babel Macro because Babel Macros doesn't support macros for custom JSX pragmas. There is a Babel macro available for `css` from `@emotion/core` if you import it from `@emotion/css/macro` though.
+All of Emotion's Babel Macros are available by adding `/macro` to the end of the import you'd normally use. (assuming you're using Create React App v2 or you've added babel-plugin-macros to your .babelrc) For example, to use the macro for styled, use `@emotion/styled/macro`.
 
 ```jsx
 import styled from '@emotion/styled/macro'
-import css from '@emotion/css/macro'
+import css from '@emotion/core/macro'
```
Comment (mitchellhamilton) [inline docs/babel-macros.mdx]:
```suggestion
import { jsx, css, Global, keyframes } from '@emotion/core/macro'
```

File: packages/babel-plugin-emotion/README.md [hunk 2]
```diff
@@ -323,6 +323,6 @@ Instead of using `babel-plugin-emotion`, you can use emotion with [`babel-plugin
 ```jsx
 import styled from 'react-emotion/macro'
```
Comment (mitchellhamilton) [inline packages/babel-plugin-emotion/README.md]:
```suggestion
```

File: packages/babel-plugin-emotion/README.md [hunk 3]
```diff
@@ -323,6 +323,6 @@ Instead of using `babel-plugin-emotion`, you can use emotion with [`babel-plugin
...
 import { css, keyframes, injectGlobal, flush, hydrate } from 'emotion/macro'
-import css from '@emotion/css/macro'
+import { css } from '@emotion/core/macro'
```
Comment (mitchellhamilton) [inline packages/babel-plugin-emotion/README.md]:
```suggestion
import { jsx, css, Global, keyframes } from '@emotion/core/macro'
```

File: packages/css/src/index.js [hunk 4]
```diff
@@ -1,10 +1,3 @@
...
-import type { Interpolation, SerializedStyles } from '@emotion/utils'
-import { serializeStyles } from '@emotion/serialize'
-
-function css(...args: Array<Interpolation>): SerializedStyles {
-  return serializeStyles(args)
-}
-
-export default css
+throw new Error(
+  "`@emotion/css` package has been deprecated and removed. Please just import `css` from `@emotion/core` - like this `import { css } from '@emotion/core'`."
```
Comment (mitchellhamilton) [inline packages/css/src/index.js]:
```suggestion
  "`@emotion/css` package has been replaced with the `css` export from `@emotion/core`. Please import it like this `import { css } from '@emotion/core'`."
```

**Comment by Andarist** [comment]
@mitchellhamilton I've applied suggested changes - anything else to do here besides adding changesets?

**Comment by mitchellhamilton** [comment]
I think changesets is the only thing

**Pull request merged**